### DISCUSSION TITLE
Directional Light

### DISCUSSION THREAD

**Discussion by Yuukichan04** [Manashiku/MMDGenshin#2] [Ideas]
is there anyway to update the shader to have a directional light so we dont need to use the light panel to control the lighting on the model? it would be better so we can put the light panel light behind the models so for ex the sun is behind the models and we can use sunflares

**Comment by Manashiku** [comment]
For some reason I didn't get an email about this discussion opening. If you use the helper plugin it does actually generate and bone you can use the offset the whole model's light direction. Unless you're specifically asking for one that is linked to the controller?

**Reply by Yuukichan04** [reply]
yea one with the controller

**Reply by Yuukichan04** [reply]
but i didnt know it did that

**Reply by Manashiku** [reply]
Next update I will add the option to control it globally for every model in the scene using the controller
